bedrock/firefox/templates/firefox/new.html

```diff
@@ -93,12 +93,15 @@
     <div class="row">
       <img src="{{ static('img/firefox/new/firefox-logo.png') }}" alt="Firefox" />
       <h1 class="large">
-      {% if l10n_has_tag('fx10_independent') %}
+      {% if LANG == 'en-US' or l10n_has_tag('spring_release') %}
+        {# L10n: Line break below for visual formatting only. #}
+        {{ _('When it’s personal,<br>choose Firefox.') }}
+      {% elif l10n_has_tag('fx10_independent') %}
         {{ _('Choose Independent.') }}
         <br>
         {{ _('Choose Firefox.') }}
       {% else %}
-        {%- if LANG in ['en-GB', 'en-US', 'en-ZA'] -%}
+        {%- if LANG in ['en-GB', 'en-ZA'] -%}
```

Comment:
I think we should drop completely this piece of conditional at this point and use ```Committed to you, your privacy and an open Web``` as fallback.